mobile/library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.h

```diff
@@ -78,19 +84,20 @@ class PlatformBridgeCertValidator : public CertValidator, Logger::Loggable<Logge
                                          PlatformBridgeCertValidator* parent);
 
   // Called when a pending verification completes. Must be invoked on the main thread.
-  void onVerificationComplete(std::thread::id thread_id, std::string hostname, bool success,
-                              std::string error_details, uint8_t tls_alert,
+  void onVerificationComplete(const Thread::ThreadId& thread_id, const std::string& hostname,
+                              bool success, const std::string& error_details, uint8_t tls_alert,
```

Comment:
this is called in a lambda, in a subsequent epoll loop, are the strings and ThreadId we capture by reference guaranteed to be in scope by the time `onVerificationComplete` is executed?

Reply:
This is called inside `postVerifyResultAndCleanUp`, which is already inside the thread routine (`verifyCertChainByPlatform`). So, all those variables will have the same lifetime as the thread routine.